src/code-templates/libraries/error-handling/error-handling.ts

```diff
@@ -1,4 +1,4 @@
-const logger = require("../../libraries/logger/logger");
+const logger = require("../logger/pino.logger");
```

Comment:
@henarbel Just a thought - Should the client care/know that we're using Pino? Woudl it be better to call this just practica-logger or logger?